README.md

```diff
@@ -1,4 +1,32 @@
 # delivery-base
 
-TODO: Enter the cookbook description here.
+Sets up the base things for a node to be use in Chef Delivery.
 
```

Comment:
We should have a requirements section here that calls out the supported platforms, minimum chef release, and any cookbooks this depends on.  See the example here: https://github.com/chef-cookbooks/keepalived/blob/master/README.md